doubly_linked_list/doubly_linked_list.py

```diff
@@ -72,9 +85,8 @@ def move_to_end(self, node):
     node.next = None
     self.tail = node 
 
-
   def delete(self, node):
-    pass
+    node.delete()
     
   def get_max(self):
-    pass
+    pass
```

Comment:
For this one you are going to need to step through the Doubly Linked List, and compare values in order to return the maximum value.  Notice that each ListNode has next & prev properties. I bet those could be useful for traversing through this data structure!